scipy/optimize/_linprog_rs.py

```diff
@@ -0,0 +1,540 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Aug 18 22:07:15 2018
+
+@author: matthaberland
+"""
+from __future__ import division, absolute_import, print_function
+import numpy as np
+from scipy.linalg import solve
+from .optimize import _check_unknown_options
+from ._bglu_dense import LU
+from ._bglu_dense import BGLU as BGLU
+from scipy.linalg import LinAlgError
+from numpy.linalg.linalg import LinAlgError as LinAlgError2
+from ._linprog_util import _postsolve
+from .optimize import OptimizeResult
+
+# TODO: cythonize c_hat = c - v.dot(A); c_hat = c_hat[~bl]
```

Comment:
If this is still relevant (doesn't seem like it would gain much), can you move it to right above the lines of code it applies to?

Reply:
Moved. I think it meant to cythonize and perform multiplication only on certain columns.